### PR TITLE
chore: add redirect for insight subdomain

### PR DESCRIPTION
**Pull request by brandonroberts**
## Description

- Adds configuration for a domain-level redirect from insight.opensauced.pizza and app.opensauced.pizza to insights.opensauced.pizza
- Docs https://docs.netlify.com/routing/redirects/redirect-options/#domain-level-redirects

## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [x] 📦 Chore (Release)
- [ ] ⏩ Revert

## Added tests?

- [ ] 👍 yes
- [ ] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [ ] 🙅 no documentation needed

### CODE REVIEW

**Comment by bdougie** [comment]
I removed the subdomain app from the original opensauced.pizza site on Netlify. I have added a 3rd alias to the site in preparation for this.

We may be able to add the app redirect in this and then make it the primary subdomain in the settings

Screen Shot 2023-09-05 at 11 13 45 PM

**Review by OgDev-01 — APPROVED** [review]
Approving to unblock merge , This is my first time seeing this implementation of redirects 🫠

Cool stuff 🍕

File: netlify.toml [hunk 1]
```diff
@@ -14,3 +14,15 @@ targetPort = 3001
...
+[[redirects]]
+  from = "https://insight.opensauced.pizza/*"
+  to = "https://insights.opensauced.pizza/:splat"
+  status = 301
+  force = true
+
+[[redirects]]
+  from = "https://app.opensauced.pizza/*"
+  to = "https://insights.opensauced.pizza/:splat"
+  status = 307
```
Comment (nickytonline) [inline netlify.toml:27]: I probably don't have the full context as to why, but any reason this one is a temporary redirect?

Comment (brandonroberts) [reply netlify.toml:27]: I made it temporary in the case that we flip them in the future, where `app` is the primary, and `insights` would then permanently redirect to `app`

**Pull request merged**